src/views/MondoMarket/index.jsx

```
@@ -0,0 +1,283 @@
+  const [isTop, setIsTop] = useState(false);
+
+  const [mondoCategories, setMondoCategories] = useState([]);
+  const [items, setItems] = useState([]);
+
+  const [topMondo, setTopMondo] = useState(null);
+  const [activeRegion, setActiveRegion] = useState(null);
+  const [currentMondo, setCurrentMondo] = useState(null);
+
+  // ** Redux ---
+  const dispatch = useDispatch();
+  const { bgColor, textColor } = useSelector((state) => state.theme);
```

ESLint

[error] 'textColor' is assigned a value but never used.

(no-unused-vars)